### ISSUE TITLE
Add 'proficiency' property to subclass object

### ISSUE THREAD

**Issue by Freedzio** [5e-bits/5e-database#258]
So

When you choose life domain for cleric, you get proficiency with heavy armor, but it is not specified in data itself, you just get a feature called "bonus proficiency", which itself is not too descriptive.

My suggestion is, maybe add `proficiencies` property to domain object? And add such property to aby subclass giving you proficiencies?

**Comment by fergcb** [comment]
A better solution imo would be to expand on the "Bonus Proficiency" feature, rather than the subclass itself. According to the SRD, the feature is what grants the proficiency, not the subclass.

Features in general could definitely do with being expanded to make them more computer friendly - e.g. using APIReferences, Choice structures, and so on.

**Comment by Freedzio** [comment]
hmm, so `proficiency` property on `feature` object? This way, the data comes from feature, bot the subclass.

Also, I agree on the idea of expanding the features. For example, it would be nice if features like "Divine Domain' or "Bard College" had some list property to choose from (despite the fact there is only one subclass choice for every class). Sometimes, to get to actual data related to a feature is tricky

**Comment by fergcb** [comment]
> hmm, so proficiency property on feature object? This way, the data comes from feature, bot (sic) the subclass.

Yes, exactly this! It would be better representative of the SRD content, and just meshes better with the semantics of the API data's structure as a result.

**Comment by bagelbits** [comment]
This is similar to #187 as well.
